modules/apps/29-fee/keeper/escrow.go

```diff
@@ -45,7 +45,7 @@ func (k Keeper) escrowPacketFee(ctx sdk.Context, packetID channeltypes.PacketId,
 }
 
 // DistributePacketFeesOnAcknowledgement pays all the acknowledgement & receive fees for a given packetID while refunding the timeout fees to the refund account.
-func (k Keeper) DistributePacketFeesOnAcknowledgement(ctx sdk.Context, forwardRelayer string, reverseRelayer sdk.AccAddress, packetFees []types.PacketFee) {
+func (k Keeper) DistributePacketFeesOnAcknowledgement(ctx sdk.Context, forwardRelayer string, reverseRelayer sdk.AccAddress, packetFees []types.PacketFee, packetID channeltypes.PacketId) error {
```

Comment:
Sorry for the confusing [comment]()! My intention was actually that by bringing in `k.DeleteFeesInEscrow` we could remove the necessity for the error to be included in the function API

The reason we don't want to return an error is because if an error is returned all state changes are reverted (including locking of the fee module). This is why we decided to bring in the deletion of the fees, because that was the only action which should not be performed if we lock the fee module

Reply:
@colin-axner I've misunderstood your idea. I just changed it and thanks you for make me better understand !